docs/en/trust.rst

```diff
@@ -381,39 +362,48 @@ giving the references of the metadata protocol for each of these.
 
-+------------------+-----------------------------+--------------+
-| Entity           | metadata type               | references   |
-+==================+=============================+==============+
-| Trust Anchor     | ``federation_entity``       | `OIDC-FED`_  |
-+------------------+-----------------------------+--------------+
-| Intermediate     | ``federation_entity``       | `OIDC-FED`_  |
-+------------------+-----------------------------+--------------+
-|                  |                             |              |
-| Wallet Provider  | ``federation_entity``       | --           |
-|                  |                             |              |
-|                  | ``wallet_provider``         |              |
-|                  |                             |              |
-|                  |                             |              |
-+------------------+-----------------------------+--------------+
-|                  |                             |              |
-| Credential Issuer| ``federation_entity``       |              |
-|                  |                             |              |
-|                  | ``openid_credential_issuer``| `OPENID4VCI`_|
-+------------------+-----------------------------+--------------+
-|                  |                             |              |
-| Relying Party    | ``federation_entity``       |              |
-|                  |                             |              |
-|                  | ``wallet_relying_party``    | `OIDC-FED`_  |
-|                  |                             |              |
-|                  |                             | `OpenID4VP`_ |
-+------------------+-----------------------------+--------------+
+.. list-table::
+   :widths: 20 20 60
+   :header-rows: 1
+
+   * - Entity
+     - Metadata type
+     - References
+   * - Trust Anchor
+     - ``federation_entity``
+     - `OIDC-FED`_
+   * - Intermediate
+     - ``federation_entity``
+     - `OIDC-FED`_
+   * - Wallet Provider
+     - ``federation_entity``, ``wallet_provider``
+     - --
+   * - Authorization Server
+     - ``federation_entity``, ``oauth_authorization_server``
+     - `OPENID4VCI`_
+   * - Credential Issuer
+     - ``federation_entity``, ``openid_credential_issuer``, [``oauth_authorization_server``]
+     - `OPENID4VCI`_
+   * - Relying Party
+     - ``federation_entity``, ``wallet_relying_party``
+     - `OIDC-FED`_, `OpenID4VP`_
+
 
 .. note::
     Wallet Provider metadata is defined in the section below.
 
     `Wallet Solution section <wallet-solution.html>`_. 
 
 
+.. note::
+    In instances where a Credential Issuer also functions as an Authorization Server, 
+    it is imperative for the Entity to incorporate both 
+    ``oauth_authorization_server`` and ``openid_credential_issuer`` within its metadata types.
+    Other implementations may divide the Credential Issuer from the Authorization Server.
```

Comment:
What does it mean practically? IMHO the PID/(Q)EAA Provider MUST have both `oauth_authorization_server` and `openid_credential_issuer`  and MAY have `openid_relying_party` or `wallet_relying_party`.

Reply:
according to openid4vci, they may have authorization_servers (ARRAY) pointing to the AS issuing access token usable for that VCI instance

Reply:
We currently don't have the `authorization_server` parameter which is optional. I'm open to consider this option but we should analyze all the implications of this in terms of security. At the current stage I suggest having both `oauth_authorization_server` and `openid_credential_issuer` Metadata in the PID/(Q)EAA Provider Entity Configuration, according to openid4vci spec:

"_If this parameter is omitted, the entity providing the Credential Issuer is also acting as the Authorization Server, i.e., the Credential Issuer's identifier is used to obtain the Authorization Server metadata._ "

Reply:
@fmarino-ipzs we cannot force the implementation in deploy multiple endpoint on the same entity. AS and RS are separated entities, therefore deploying them separately in different entity, according to the openid specs, has sense.